shaders/src/light_indirect.fs

```diff
@@ -59,6 +78,8 @@ vec3 Irradiance_SphericalHarmonics(const vec3 n) {
         + frameUniforms.iblSH[8] * (n.x * n.x - n.y * n.y)
 #endif
         , 0.0);
+
+    return TintIbl(rawResult);
```

Comment:
In theory we should be able to apply the tinting on the SH themselves.

Reply:
Unfortunately, overlay blend is not linear (`OverlayBlend(sum_i coeff[i](**n**) * SH[i], c) != sum_i coeff[i](**n**) * OverlayBlend(SH[i], c)`), so we can't do that (theoretically speaking). I would have preferred a simple multiplicative blend for this very reason but we couldn't make ends meet visually with those, we had to let the bright parts keep their un-tinted color, which led to the nonlinear overlay blend. Now, in practice, it may not matter much, we can do a comparison if you'd like to.

In terms of always paying for this: in general, this might indeed be better as a material option/permutation.